src/azul/queues.py

```diff
@@ -220,9 +220,13 @@ def wait_to_stabilize(self) -> int:
         Wait for the work queues to reach a steady state.
         """
         sleep_time = 10
+        # Partitions that initially fail are processed by the retry lambda, this
+        # notifications sit in the queue for 15 min before being picked up. The
+        # timeout should account for that occasional delay.
```

Comment:
```suggestion
        # Indexing can still succeed after a transient stall. A stall's 
        # transience cannot be proven until all lambdas and their respective
        # retries repeatedly time out, but this would result in an unreasonably
        # long wait time. Waiting for just one retry is sufficient to
        # accommodate the most probable scenarios for transient stalls.
```